app/scripts/lib/ppom/ppom-middleware.test.ts

```diff
@@ -205,6 +207,35 @@ describe('PPOMMiddleware', () => {
     expect(validateRequestWithPPOM).not.toHaveBeenCalled();
   });
 
+  it('does not do validation for SIWE signature', async () => {
+    const middlewareFunction = createMiddleware({
+      securityAlertsEnabled: false,
```

Comment:
Shouldn't this set to be `true` in order to bypass early return?

Reply:
Good catch, I updated the PR.

Reply:
Good point, I updated PR again